.travis.yml

```diff
@@ -63,7 +63,7 @@ install:
   - git clone https://github.com/openresty/openresty.git ../openresty
   - git clone https://github.com/openresty/openresty-devel-utils.git
   - git clone https://github.com/simpl/ngx_devel_kit.git ../ndk-nginx-module
-  - git clone https://github.com/openresty/lua-nginx-module.git ../lua-nginx-module
+  - git clone -b feat/ngx_http_lua_ffi_str_replace_char https://github.com/thibaultcha/lua-nginx-module.git ../lua-nginx-module
```

Comment:
Direct link to the FFI API on my fork: https://github.com/thibaultcha/lua-nginx-module/commit/51505338fbb5df261df670a75a2c4aab748e6b3c